transports/jms/src/test/java/org/mule/transport/jms/integration/JmsTemporaryReplyToTestCase.java

```diff
@@ -22,43 +24,81 @@
  */
 public class JmsTemporaryReplyToTestCase extends AbstractJmsFunctionalTestCase
 {
+
+    private static String ECHO_FLOW_NAME = "EchoFlow";
+
     @Override
     protected String getConfigFile()
     {
         return "integration/jms-temporary-replyTo.xml";
     }
 
+    // Fixed Reply Queue
```

Comment:
Can you avoid these kind of "test group separator" comments and improve test names instead?